tests/api/content/valid/minimum.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../basic.rng"?>
+
+<content xmlns="http://libero.pub">
```

Comment:
I greatly fear `<content>` becoming overloaded with the [`content:` key currently in use](https://github.com/elifesciences/api-raml/blob/develop/src/model/blog-article.v1.yaml#L7). Evaluated alternatives like `<parts>` or anything not in use right now?

Reply:
In the grammar, it also clashes with the `attributes` vs `content` separation leading to `libero.content.content`